community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactions.java

```diff
@@ -202,6 +205,24 @@ protected void dispose( KernelTransactionImplementation tx )
     }
 
     /**
+     * Give an approximate set of all transactions currently running together with associated metadata as
+     * computed by the provided selector function.
+     * This is not guaranteed to be exact, as transactions may stop and start while this set is gathered.
+     *
+     * @return the (approximate) set of open transactions.
+     */
+    public <T> Set<Pair<KernelTransactionHandle, T>> activeTransactions(
```

Comment:
do not think this method should be introduced in KernelTransaction at all:
 it's copy paste of active transactions method, it should not return too generic pair thingy, used only in build in procedures so mapping, if its necessary can be produced there